events/room-booking.md

```diff
@@ -1,33 +1,49 @@
 # Room Booking
+Steps for room and AV (Audio Visual) cell booking for conducting workshops and other events.
 
 ## Steps for Room Booking
-
-- Go to the Junior Executive, Academic Section (UG) (Mr Shouvik Adhikari as of now) and get the form for room booking.
+- Go to the Junior Executive, Academic Section (UG) (Mr. Shouvik Adhikari as of now) and get the form for room booking. [Here](../assets/room-booking-form.pdf) is a scan of the form.
 - This form is only for room booking and not for AV support.
-- Fill in the details necessary for booking a room. Date and time, Duration.
-- Would need the signature of a Professor and the HOD of the respective department as the booking is made in the professor's name. (The toughest part of the job). CSE professors are easily approachable for this, so can reach them out.
-- Submit this form back to the Jr. Executive, and he would reach back in a day or two for the payment.
+- Fill in the details necessary for booking a room. Date, time, and duration.
+- Get the signature of a Professor and the HOD of the respective department as the booking is made in the professor's name (Approach professors at the earliest). CSE professors are easily approachable for this, so can reach them out.
+- Submit this form back to the Jr. Executive, and they would reach back in a day or two for the payment.
 - The Jr. Executive would give a draft to be paid at the fees counter and take the payment receipt.
```

Comment:
- Mention brief location of fees section :) it's literally bhool bhulaiyya

Reply:
Where is it?

Reply:
good question, @chirag-ghosh thoda sa idea dedo pls 
At bare minimum mention that it is in main building itself

Reply:
Establishment Section, 2nd Floor, Main Building